components/runtime-models/src/internal/member.rs

```diff
@@ -29,6 +29,10 @@ pub struct UpdateGuildMemberFields {
     #[ts(optional)]
     #[ts(type = "string[]")]
     pub roles: Option<Vec<Id<RoleMarker>>>,
+
+    #[ts(optional)]
+    #[ts(type = "number|null")]
+    pub communication_disabled_until: Option<NotBigU64>,
```

Comment:
Use a Option<Option<...>> here togheter with  `#[serde(deserialize_with = "crate::deserialize_optional_field")]` 

In case you don't know what it means, the outer option signifies whether we are updating the field at all, e.g if the outer option is none it's not changed and the original value is kept, the inner option is the new value, which if None will remove the timeout

Reply:
on the ts side this is `undefined | null | number`, undefined being unchanged, null removing the timeout and so on

Reply:
>Use a Option<Option<...>> here togheter with `#[serde(deserialize_with = "crate::deserialize_optional_field")]`
>
>In case you don't know what it means, the outer option signifies whether we are updating the field at all, e.g if the outer option is none it's not changed and the original value is kept, the inner option is the new value, which if None will remove the timeout

Ah right, not sure how I missed that lol

> on the ts side this is `undefined | null | number`, undefined being unchanged, null removing the timeout and so on

This is implied by TypeScript `?:`